features/step_definitions/admin_steps.rb

```diff
@@ -150,6 +150,11 @@ def update_settings(settings)
   click_button("Update")
 end
 
+Then /^the user "([^\"]*)" should be permanently banned$/ do |user|
```

Comment:
Ambiguous regexp literal. Parenthesize the method arguments if it's surely a regexp literal, or add a whitespace to the right of the `/` if it should be a division.